CHANGELOG.md

```diff
@@ -14,6 +14,10 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 - Add client metric support to `go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp`. (#4707)
 - Add peer attributes to spans recorded by `NewClientHandler`, `NewServerHandler` in `go.opentelemetry.io/contrib/instrumentation/google.golang.org/grpc/otelgrpc`. (#4873)
 
+### Changed
+
+- Updated jaeger version in `go.opentelemetry.io/contrib/samplers/jaegerremote/example` example. (#4892)
```

Comment:
My bad, I didn't fully read the other changes when I suggested the changelog entry. You're not upgrading the jaeger version, you're updating the collector.

```suggestion
- Updated otel-collector from `otel/openteleemtry-collector` to `otel/opentelemetry-collector-contrib` in `go.opentelemetry.io/contrib/samplers/jaegerremote/example` example. (#4892)
```